spec/factories/factories.rb

```diff
@@ -48,7 +48,7 @@
     unit_price 15050
   end
 
-  factory :transactions do
+  factory :transaction do
```

Comment:
I've had to make this same change to make my tests work. Hopefully it won't create merge conflicts